app/components/preprint-form-project-select.js

```diff
@@ -116,24 +89,28 @@ export default Component.extend(Analytics, {
             this.attrs.clearDownstreamFields('belowNode');
             this.set('selectedNode', node);
             this.set('osfProviderLoaded', false);
-            this.send('changeExistingState', this.get('_existingState').CHOOSE);
             this.get('selectedNode.files').then(this._setStorageProvider.bind(this));
-            this.attrs.nextUploadSection('chooseProject', 'chooseFile');
+            this.attrs.nextUploadSection('chooseProject', 'selectExistingFile');
             this.get('metrics')
                 .trackEvent({
                     category: 'dropdown',
                     action: 'select',
                     label: 'Submit - Choose Project',
                     extra: node.id,
                 });
-            this.getNodePreprints(node);
-            this.getContributors(node);
+            // When project is selected, its title is pre-populated as the preprint's title
+            this.set('title', node.get('title'));
+            this.set('titleValid', true);
+            // The project's contributors are loaded.  They will be copied to the preprint
+            // upon "Save and Continue"
+            this.getProjectContributors(node);
         },
+
         selectFile(file) {
-            // Select existing file from file-browser
-            this.attrs.clearDownstreamFields('belowFile');
```

Comment:
We don't need to clear downstream fields below the file any longer now that we only have one option when submitting your preprint.  Selecting a new file no longer needs to clear the title downstream.